samples/DevTestLabs/GatewaySample/arm/environment/WindowsJumpboxToTwoLinuxVMs/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "itemDisplayName": "Windows Jumpbox To Two Linux VMs",
+  "description": "This template Windows VM with two Linux VMs on an DTL subnet.",
```

Comment:
Re-word the `description`. It seems to be an incomplete sentence. It would be good to describe why we need such an environment. What's the value of the sample environment?